.infrastructure/helm-chart/todoapp/charts/mysql/templates/statefulSet.yml

```diff
@@ -0,0 +1,91 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ .Chart.Name }}-stateful-set
+  namespace: {{ .Values.common.namespace }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ .Values.mysql.labelName }}
+  replicas: {{ .Values.statefulSet.replicas }}
+  serviceName: {{ .Chart.Name }}-service
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.mysql.labelName }}
+    spec:
+      containers:
+      - name: {{ .Chart.Name }}-container
+        image: {{ .Values.mysql.image.name }}:{{ .Values.mysql.image.tag }}
+        env:
+        {{- range $k, $v := .Values.secret }}
+        - name: {{ $k }}
+          valueFrom:
+            secretKeyRef:
+              name: {{ $.Chart.Name }}-secret
+              key: {{ $k }}
+        {{- end }}
+        ports:
+        - name: mysql
+          containerPort: 3306
+        volumeMounts:
+        - name: data
+          mountPath: /var/lib/mysql
+          subPath: mysql
+        - name: config-map
+          mountPath: /docker-entrypoint-initdb.d
+        resources:
+          requests:
+            cpu: {{ .Values.statefulSet.resources.requests.cpu }}
+            memory: {{ .Values.statefulSet.resources.requests.memory }}
```

Comment:
Ensure that `.Values.statefulSet.resources.requests.cpu` and `.Values.statefulSet.resources.requests.memory` are defined in your `values.yaml` file to specify resource requests for the container.